build.sbt

```diff
@@ -94,6 +94,7 @@ lazy val ossFileSystem = (project in file("filesystems/oss"))
 
 lazy val databaseSql = (project in file("database/sql"))
   .withLibrarySettings("cromwell-database-sql", databaseSqlDependencies)
+  .dependsOn(core)
```

Comment:
Necessary to let `WorkflowMetadataSummaryEntryComponent`  see the `WorkflowMetadataKeys`